extensions/blocks/podcast-player/style.scss

```diff
@@ -403,35 +401,48 @@ $jetpack-podcast-player-error: $alert-red;
 		.mejs-button > button,
 		.jetpack-podcast-player__track-status-icon--playing {
 			background-image: none;
-			background-color: var( --jetpack-podcast-player-primary );
-			mask: url( '/wp-includes/js/mediaelement/mejs-controls.svg' );
+			outline-color: $jetpack-podcast-player-primary;
+			outline-color: var( --jetpack-podcast-player-primary );
```

Comment:
Should we also set `outline-style: solid`? currently, we have mixed styles between different components and something like this might make it more consistent. leaving up to you

Reply:
This is something I'd leave for another PR where we apply focus styles from design (which AFAIR we don't have yet). For example, links in TwentyTwenty don't use an outline at all - they use blue box-shadow, so we'd have to account for that, too.

Reply:
This is how the TwentyTwenty's link's box-shadow looks on a blue background:

![image](https://user-images.githubusercontent.com/1451471/80224192-a9ef7a80-8649-11ea-817a-99ebd333ca11.png)

Reply:
I guess the simplest and least invasive solution (until we have a design in-place) could be to reset box-shadow on a focused link and define our own outline, e.g. `1px solid $secondary`. This would make sure the outline is always visible and consistent throughout the Player's UI. Example screenshot:

![image](https://user-images.githubusercontent.com/1451471/80224726-65181380-864a-11ea-8b80-271326af1b7b.png)

Reply:
Calling in reinforcements. @ItsJonQ any recommendations on how we keep theme specific styles from bleeding into this block? Should we be doing a full reset? If we do that, won't it affect global styles? I'm not entirely sure how this is all _supposed_ to work.

Reply:
`1px solid $secondary` looks good to me and makes sense. Happy to keep it separate and ship this improvement first if you want to

Reply:
This gets tricky fast. I think it's ideal to let the theme (or browser, if no styles are set) handle the focus style and we not reset the focus at all. That said, I see how the blue on blue would be an issue. I think this would be fairly rare for the focus style color to match the background though.

I think using the secondary color is a clever idea to avoid the clash. I'm going to see if I can find any recommendations on if it's OK to change focus style throughout a page.

Reply:
This article from Deque is saying focus indicator _does not need to be the same for all elements_: https://www.deque.com/blog/give-site-focus-tips-designing-usable-focus-indicators/

> Good contrast
Complementary shape and size to the element
Color scheme is complementary, but also stands out
Doesn’t need to be the same for all elements
Animations help users track focus movement
Should degrade gracefully (be visible on older browsers)
Should be the same across browsers

I think secondary color would be a good solution then, if that's OK with Global Styles.

Reply:
This should be a small change TBH. I will do some testing with some themes & Global Styles and, if everything looks good, I'll push the change here. How does that sound?

Reply:
Just pushed the outline - it's ready for testing. Some ME.js elements could still look better with that outline but I didn't want to start restyling them here.

This is how it looks now (Varia + Global Styles):

![Apr-24-2020 17-31-10](https://user-images.githubusercontent.com/1451471/80230384-2b4b0b00-8652-11ea-8e65-3297e6f3c7ad.gif)

What do you think?

Reply:
I appreciate the consistency a lot and can't think of a reason not to do it but I'll defer to @jeryj, since he's more skilled in this area.

Reply:
Looks good to me.

Reply:
I chatted with @ItsJonQ and he suggested doing overrides where needed, but limiting them as much as possible. I think in this case it makes sense.

Reply:
I think this is a good solution 👍

Reply:
:shipit: thanks y'all!